vllm_detector_adapter/api_server.py

```diff
@@ -314,8 +315,8 @@ def add_chat_detection_params(parser):
     parser = FlexibleArgumentParser(
         description="vLLM OpenAI-Compatible RESTful API server."
     )
-    parser = make_arg_parser(parser)
-
+    # convert to our custom env var arg parser
+    parser = LocalEnvVarArgumentParser(parser=make_arg_parser(parser))
```

Comment:
should this be changed in `vllm_detector_adapter/start_with_tgis_adapter.py` as well ?